CheatSheets/git-cheatsheet.md

```diff
@@ -124,6 +126,8 @@ created: 2022-10-18
 | `git bisect start` | Starts the bisection search process to find that bad commit which introduced the bug we're facing right now |
 | `git bisect good` <br /> `git bisect good [Commit ID]` | Takes up the good commit, which is that one where the bug was not there |
 | `git bisect bad [Commit ID]` | Takes up the bad commit, which is that one where the bug was there. If commit ID is not provided, then it takes up the current commit as the bad commit |
+| `git grep` | Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in given tree objects |
```

Comment:
```suggestion
| `git grep "hello"` |  - A text search on all files in the directory <br /> - Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in the given tree objects |
```
| Command | Description |
| ------------ | --------------- |
| `git grep "hello"` |  - A text search on all files in the directory <br /> - Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in the given tree objects |